config.development.yaml

```diff
@@ -12,6 +12,7 @@ services:
       recaptcha_sitekey: 6Lc2BtYUAAAAAOUBI-9r1sDJUIfG2nt6C43noOXh
       db:
         connection_uri: MONGODB_URI
+        type: mongodb
```

Comment:
issue: these configs are not used anymore, you need to change the pkl files in the deployment folder